### DISCUSSION TITLE
GraphQL: Sorting ASC / DESC

### DISCUSSION THREAD

**Discussion by tarwin** [directus/directus#6015] [Archive]
It doesn't seem that the GraphQL endpoint supports sorting other than `ASC`. The current API seems to only support a single string, or array of strings for `sort`, and this is directly turned into something like this:

`article(sort: "createdAt")`
`SELECT createdAt FROM table ORDER BY table.createdAt ASC`

Reading about GraphQL it seems that the way other do it is using an object ie:

`article(orderBy: { createdAt: asc })`

**Comment by rijkvanzanten** [comment]
It does, see https://docs.directus.io/reference/api/query/#sort 👍🏻

**Reply by tarwin** [reply]
Oh god. How'd I miss that?! Thank you.
